From e94cd423e82470e1da183ee117bf673fa78bd3f1 Mon Sep 17 00:00:00 2001
From: Jason Zeng <jason.zeng@intel.com>
Date: Mon, 15 Aug 2022 16:58:14 +0800
Subject: [PATCH 0461/1646] [Backport] Enable default config for Intel NTB LTR
 vendor support for gen4 NTB

Intel inclusion
category: feature
bugzilla: https://gitee.com/openeuler/intel-kernel/issues/I5MJBR

Reference: https://gitee.com/openeuler/kernel/commit/74db7ea6b45761217dbce0fd68a7825720f7769f

--------------------------------------------

Enable necessary kernel configs for Intel NTB LTR vendor support for gen4 NTB in openeuler_defconfig.

Reviewed-by: Wu Zheng <wu.zheng@intel.com>
Signed-off-by: Jason Zeng <jason.zeng@intel.com>
Signed-off-by: Aichun Shi <aichun.shi@intel.com>
Signed-off-by: Jialin Zhang <zhangjialin11@huawei.com>
---
 arch/x86/configs/openeuler_defconfig | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 4b65464447ed..80ac17cf1fad 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -2608,6 +2608,7 @@ CONFIG_NETCONSOLE=m
 CONFIG_NETCONSOLE_DYNAMIC=y
 CONFIG_NETPOLL=y
 CONFIG_NET_POLL_CONTROLLER=y
+CONFIG_NTB_NETDEV=m
 CONFIG_TUN=m
 CONFIG_TAP=m
 # CONFIG_TUN_VNET_CROSS_LE is not set
@@ -7145,10 +7146,10 @@ CONFIG_NTB=m
 # CONFIG_NTB_IDT is not set
 CONFIG_NTB_INTEL=m
 # CONFIG_NTB_SWITCHTEC is not set
-# CONFIG_NTB_PINGPONG is not set
-# CONFIG_NTB_TOOL is not set
-# CONFIG_NTB_PERF is not set
-# CONFIG_NTB_TRANSPORT is not set
+CONFIG_NTB_PINGPONG=m
+CONFIG_NTB_TOOL=m
+CONFIG_NTB_PERF=m
+CONFIG_NTB_TRANSPORT=m
 # CONFIG_VME_BUS is not set
 CONFIG_PWM=y
 CONFIG_PWM_SYSFS=y
-- 
2.17.1

